bench/dashboard/styles.css

```
@@ -0,0 +1,251 @@
+  gap: 6px;
+}
+
+.header-links a:hover {
+  color: var(--text);
+  border-color: var(--border-light);
+}
+
+.header-links a svg {
+  width: 13px;
+  height: 13px;
+  fill: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)